main.go

```diff
@@ -63,6 +63,7 @@ import (
 	runtimecatalog "sigs.k8s.io/cluster-api/exp/runtime/catalog"
 	runtimecontrollers "sigs.k8s.io/cluster-api/exp/runtime/controllers"
 	runtimehooksv1 "sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
+	expapiwebhooks "sigs.k8s.io/cluster-api/exp/webhooks"
```

Comment:
nit: let's go with expwebhooks as import alias (same in environment.go)

Reply:
Done